pkgdb/src/buildenv/command.cc

```diff
@@ -86,9 +91,30 @@ BuildEnvCommand::run()
   auto store = this->getStore();
   auto state = this->getState();
 
+  if ( this->storePath.has_value() && this->outLink.has_value() )
+    {
+      std::filesystem::path path( this->storePath.value() );
+      nix::StorePath        storePath( std::string( path.filename() ) );
```

Comment:
Fixed. I just did a `std::filesystem::exists` instead of `store->ensurePath` because `ensurePath` will try to build/substitute and we explicitly don't want to do that here.